pipeline/notifications/rating_reminder.py

```diff
@@ -0,0 +1,33 @@
+from typing import List
+
+from pipeline.notifications.notifier import Notifier
+from pipeline.types import NotificationInfo, NotificationInput, NotificationType
+
+
+class RatingReminderNotifier(Notifier):
+    def __init__(self):
+        pass
+
+    def get_notifications(
+        self, inp: NotificationInput
+    ) -> List[NotificationInfo]:
+        remind_users = []
+        min_chat: int = 3
+        numMessages = 0
+        listUsers = inp.chatdata.participants
+        chatMessages = inp.chatdata.messages
+        dateTime = inp.run_datetime
+        for messagesSent in chatMessages:
+            if messagesSent.from_user in listUsers:
+                numMessages += 1
+        if dateTime.weekday() == 5 and numMessages >= min_chat:
+            for uid, user in listUsers.items():
```

Comment:
This works! But the logic is not quite correct. We should only give notifications to users who sent a certain number of messages, that way if a user never messaged the chat, we don't ask them to rate the match.

What this logic does is check if the total number of messages is above a certain number, the send a notification to all users in the chat, regardless of whether or not they participated.

We can merge this and it can be a future enhancement to change the functionality!